lib/events.js

```diff
@@ -140,6 +146,46 @@ ObjectDefineProperty(EventEmitter, 'defaultMaxListeners', {
   }
 });
 
+ObjectDefineProperties(EventEmitter, {
+  kMaxEventTargetListeners: {
+    value: kMaxEventTargetListeners,
+    enumerable: false,
+    configurable: false,
+    writable: false,
+  },
+  kMaxEventTargetListenersWarned: {
+    value: kMaxEventTargetListenersWarned,
+    enumerable: false,
+    configurable: false,
+    writable: false,
+  },
+  defaultEventTargetMaxListeners: {
+    get() { return defaultEventTargetMaxListeners; },
+    enumerable: false,
+    configurable: false,
+  }
+});
+
+EventEmitter.setEventTargetMaxListeners =
+  function(n = defaultEventTargetMaxListeners, ...eventTargets) {
+    if (typeof n !== 'number' || n < 0 || NumberIsNaN(n))
+      throw new ERR_OUT_OF_RANGE('n', 'a non-negative number', n);
+    if (eventTargets.length === 0) {
+      defaultEventTargetMaxListeners = n;
+    } else {
+      // Performance for forEach is now comparable with regular for-loop
```

Comment:
That's... cool to know though I doubt this is a performance sensitive spot :]

Reply:
Perhaps, perhaps not. The comment is good for folks coming in later and digging around since we've often steered people away from using forEach.